relay-server/src/services/stats.rs

```diff
@@ -14,13 +15,19 @@ use crate::statsd::{RelayGauges, TokioGauges};
 pub struct RelayStats {
     config: Arc<Config>,
     upstream_relay: Addr<UpstreamRelay>,
+    redis_pool: Option<RedisPool>,
```

Comment:
Would this work, instead of silencing the unused warning?

```suggestion
    #[cfg(feature = "processing")]
    redis_pool: Option<RedisPool>,
```

Reply:
Yes but then I have even more `#[cfg(processing)]` with conditional arguments. But can do that!

Reply:
Only if it simplifies things, otherwise feel free to ignore.